pkg/plugin/aggregation/status.go

```diff
@@ -123,3 +139,28 @@ func GetStatus(client kubernetes.Interface, namespace string) (*Status, error) {
 
 	return &status, nil
 }
+
+// SetStatus sets the status on the pod via an annotation. It will overwrite the
+// existing status.
+func SetStatus(client kubernetes.Interface, namespace string, status *Status) error {
```

Comment:
Perhaps this should include Pod in its name, `SetPodStatus`?

Reply:
👍 I'll fix up the name. Yes the term status is very overloaded. I had just copied this from elsewhere and I think it may be able to be removed from the diff now; I'll double check. Either way I'll clarify the name.